app/code/core/Mage/Adminhtml/controllers/Sales/Order/CreditmemoController.php

```diff
@@ -310,7 +320,7 @@ public function saveAction()
                 $creditmemo->getOrder()->setCustomerNoteNotify(!empty($data['send_email']));
                 $this->_saveCreditmemo($creditmemo);
                 $creditmemo->sendEmail(!empty($data['send_email']), $comment);
-                $this->_getSession()->addSuccess($this->__('The credit memo has been created.'));
+                $this->_getSession()->addSuccess($this->__('The credit memo has been created.') . $succesubscribemsg);
```

Comment:
I've not seen this pattern anywhere else in Magento. There is no harm in calling `addSuccess` multiple times.